test/v1beta1/occurrence_test.go

```diff
@@ -12,6 +15,12 @@ import (
 	"testing"
 )
 
+type occurrenceFilterTestCase struct {
```

Comment:
You could inline this, or declare within function scope to not have as specific of a name.